src/OpenTelemetry.Contrib.Instrumentation.Wcf/TelemetryClientMessageInspector.cs

```diff
@@ -111,6 +110,8 @@ public object BeforeSendRequest(ref Message request, IClientChannel channel)
                     {
                         activity.SetTag(WcfInstrumentationConstants.SoapViaTag, request.Properties.Via.ToString());
                     }
+
+                    WcfInstrumentationActivitySource.Options.Enrich?.Invoke(activity, WcfEventNames.BeforeSendRequest, request);
```

Comment:
Would you mind wrapping these calls in try/catch? Example here:

https://github.com/open-telemetry/opentelemetry-dotnet/blob/5ddf9a486e755c53ab73debf87286a934fcbbb51/src/OpenTelemetry.Instrumentation.AspNet/Implementation/HttpInListener.cs#L152-L159

Reply:
Done.  Also added same exception logging to `TelementryDispatchMessageInspector`